nodeup/pkg/model/docker.go

```diff
@@ -557,7 +557,7 @@ var dockerVersions = []dockerVersion{
 		ExtraPackages: map[string]packageInfo{
 			"container-selinux": {
 				Version: "2.68",
-				Source:  "http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.68-1.el7.noarch.rpm",
+        Source:  "http://mirror.centos.org/centos/7.6.1810/extras/x86_64/Packages/container-selinux-2.68-1.el7.noarch.rpm",
```

Comment:
It looks like your editor may have replaced the tabs with spaces. Can you update your commit to use tabs?

Reply:
I pushed a fix, so this should be resolved shortly.

Reply:
Thanks! Sorry about that.